src/main/java/com/google/sps/servlets/Statistics.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Statistics {
+  private List<Double> aggregateValues;
+  private Double averageScore;
+
+  public Statistics(List<Double> aggregateValues, double averageScore) {
```

Comment:
Do we use double or Double?

Reply:
Both - list of Double objects and double vars

Reply:
TBh I actually dont know if that's correct actually lemme check

Reply:
i would choose Double for avg as well.... since we are doing avg calculation of a list of Double

Reply:
Is there a reason to use Double? If the primitive type is fine then we should use double.